code/game/machinery/computer/buildandrepair.dm

```diff
@@ -462,6 +462,35 @@
 		drop_computer_parts()
 	return ..() // will qdel the frame
 
+/obj/structure/computerframe/verb/rotate()
```

Comment:
I'd really like to not use verbs anymore if it can be avoided; just having an alt click is good enough.